packages/esm-outpatient-app/src/patient-info/patient-info.scss

```diff
@@ -43,3 +51,9 @@
     min-height: 2rem;
   }
 }
+
+.patientEditBtn {
+  float: inline-end;
```

Comment:
The parent `div` is a flex container with `justify-content` set to `space-between`. So by default, the `Edit patient` button will be packed toward the end edge of the alignment container in the main axis. So, this `float` property is not required.